optimizely/decision_service.py

```diff
@@ -424,14 +462,19 @@ def get_variation_for_rollout(self, project_config, feature, user):
 
         return Decision(None, None, enums.DecisionSources.ROLLOUT), decide_reasons
 
-    def get_variation_for_feature(self, project_config, feature, user_context, options=None):
+    def get_variation_for_feature(
+        self,
+        project_config: ProjectConfig,
+        feature: entities.FeatureFlag,
+        user_context: OptimizelyUserContext,
+        options: Optional[Sequence[str | OptimizelyDecideOption]] = None
```

Comment:
This is my mistake, this can actually only be `str`. `OptimizelyDecideOption` is not a real enum, so it can't be used as the type (without being converted).